docs/Parameters-Tuning.rst

```diff
@@ -36,15 +36,143 @@ To get good results using a leaf-wise tree, these are some important parameters:
+
+Increase ``min_gain_to_split``
+******************************
+
+When adding a new tree node, LightGBM chooses the split point that has the largest gain. Gain is basically the reduction in training loss that results from adding a split point. By default, LightGBM sets ``min_gain_to_split`` to 0.0, which means "there is no improvement that is too small". However, in practice you might find that very small improvements in the training loss don't have a meaningful impact on the generalization error of the model. Increase ``min_gain_to_split`` to reduce training time.
+
+Increase ``min_data_in_leaf`` and ``min_sum_hessian_in_leaf``
+*************************************************************
+
+Depending on the size of the training data and the distribution of features, it's possible for LightGBM to add tree nodes that only describe a small number of observations. In the most extreme case, consider the addition of a tree node that only a single observation from the training data falls into. This is very unlikely to generalize well, and probably is a sign of overfitting.
+
+This can be prevented indirectly with parameters like ``max_depth`` and ``num_leaves``, but LightGBM also offers parameters to help you directly avoid adding these overly-specific tree nodes.
+
+- ``min_data_in_leaf``: Minimum number of observations that must fall into a tree node for it to be added.
+- ``min_sum_hessian_in_leaf``: Minimum sum of the Hessian (second derivative of the objective function evaluated for each observation) for observations in a leaf. For some regression objectives, this is just the minimum number of records that have to fall into each node. For classification objectives, it represents a sum over a distribution of probabilities. See `this Stack Overflow answer <https://stats.stackexchange.com/questions/317073/explanation-of-min-child-weight-in-xgboost-algorithm>`_ for a good description of how to reason about values of this parameter.
+
+Grow Less Trees
+'''''''''''''''
+
+Decrease ``num_iterations``
+***************************
+
+The ``num_iterations`` parameter controls the number of boosting rounds that will be performed. Since LightGBM uses decision trees as the learners, this can also be thought of as "number of trees".
+
+Choosing the right value of ``num_iterations`` is highly dependent on the data and objective, so this parameter is often chosen from a set of possible values through hyperparameter tuning.
+
+Decrease ``num_iterations`` to reduce training time.
+
+Use Early Stopping
+******************
+
+If early stopping is enabled, after each boosting round the model's performance is evaluated against a validation set that contains data not available to the training process. That performance is then compared to the performance as of the previous boosting round. If the model's performance fails to improve for some number of consecutive rounds, LightGBM stops the training process.
```

Comment:
Similar to comment above, maybe clearer to use "accuracy" instead of "performance".

Reply:
makes sense. I just replaced all of these with "accuracy" or "training accuracy": https://github.com/microsoft/LightGBM/pull/3628/commits/d615ad46df1c07eba298aab7d37ad95f1d513b1c